plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -398,27 +411,30 @@ private TestExecutionResult testPluginAgainst(MavenCoordinates coreCoordinates,
                     System.out.println("Deleting working directory "+pluginCheckoutDir.getAbsolutePath());
                     FileUtils.deleteDirectory(pluginCheckoutDir);
                 }
+
                 pluginCheckoutDir.mkdir();
                 System.out.println("Created plugin checkout dir : "+pluginCheckoutDir.getAbsolutePath());
 
                 if (localCheckoutProvided()) {
                     if (!onlyOnePluginIncluded()) {
-                        throw new RuntimeException("You specified a local clone but did not choose only one plugin to execute PCT against it");
+                        File localCheckoutPluginDir = new File(config.getLocalCheckoutDir(), plugin.name);
+                        File pomLocalCheckoutPluginDir = new File(localCheckoutPluginDir, "pom.xml");
```

Comment:
One doubt: in case the folder did not exist, the File object would still be created?

Reply:
Yes, it is created, but when exists method is called, it returns `false`

Reply:
Great! Thanks for the clarification.